client/routes/workflow/query.vue

```diff
@@ -18,16 +18,30 @@
         Run
       </a>
     </header>
-    <pre v-if="queryResult && queryResult.payloads">{{ queryResult.payloads }}</pre>
+    <pre v-if="queryResult && queryResult.payloads">{{
+      queryResult.payloads
+    }}</pre>
     <span class="error" v-if="error">{{ error }}</span>
-    <span class="no-queries" v-if="queries && queries.length === 0">
+    <span v-if="!isWorkerRunning">
+      No worker is running for the task queue 
```

Comment:
Consider "There is no Worker currently listening to the Task Queue" ("listening to" and "running for" maybe interchangeable)
Make sure to capitalize Worker & Task Queue

Reply:
that is a much better wording!

Reply:
My b. Probably with "are no Workers" i.e. 

"There are no Workers currently listening to the Task Queue: "

Reply:
done